Spoony-iOS/Spoony-iOS/Source/Feature/Home/Components/BottomSheet/CustomBottomSheet.swift

```diff
@@ -0,0 +1,162 @@
+                withAnimation(.spring(response: 0.2, dampingFraction: 0.8, blendDuration: 0)) {
+                    dragOffset = 0
+                    isPresented = false
+                }
+            } else {
+                withAnimation(.spring(response: 0.2, dampingFraction: 0.8, blendDuration: 0)) {
+                    dragOffset = 0
+                }
+            }
+            return
+        }
+        
+        let newHeight = currentHeight - value.translation.height
+        let velocityThreshold: CGFloat = 300
+        
+        withAnimation(.spring(response: 0.2, dampingFraction: 0.8, blendDuration: 0)) {
+            if verticalVelocity > velocityThreshold {
+                // 빠르게 아래로 스와이프
+                if currentHeight <= BottomSheetStyle.half.height {
+                    isPresented = false
+                } else {
+                    currentHeight = BottomSheetStyle.half.height
+                }
+            } else if verticalVelocity < -velocityThreshold {
+                // 빠르게 위로 스와이프
+                currentHeight = BottomSheetStyle.full.height
+            } else {
+                // 일반적인 드래그
+                if value.translation.height > 0 {
+                    if newHeight < BottomSheetStyle.minimal.height + 50 {
+                        isPresented = false
+                    } else if newHeight < BottomSheetStyle.half.height {
+                        currentHeight = BottomSheetStyle.minimal.height
+                    } else if newHeight < BottomSheetStyle.full.height {
+                        currentHeight = BottomSheetStyle.half.height
+                    }
+                } else {
+                    if newHeight > BottomSheetStyle.half.height + 50 {
+                        currentHeight = BottomSheetStyle.full.height
+                    } else if newHeight > BottomSheetStyle.minimal.height + 50 {
+                        currentHeight = BottomSheetStyle.half.height
+                    } else {
+                        currentHeight = BottomSheetStyle.minimal.height
+                    }
+                }
+            }
+            dragOffset = 0
+        }
+    }
+    
+    var body: some View {
+        GeometryReader { geometry in
+            ZStack(alignment: .bottom) {
+                if isPresented {
+                    VStack(spacing: 0) {
+                        RoundedRectangle(cornerRadius: 3)
+                            .fill(Color.gray)
+                            .frame(width: 40, height: 5)
+                            .padding(.top, 8)
+                        
+                        content
+                    }
+                    .frame(height: currentHeight - dragOffset)
+                    .background(Color.white)
+                    .cornerRadius(20, corners: [.topLeft, .topRight])
+                    .gesture(
+                        DragGesture()
+                            .updating($isDragging) { value, state, _ in
```

Comment:
```suggestion
                            .updating($isDragging) { _, state, _ in
```

안쓰는 클로저 파라미터는 와일드카드(_)로 표현 해도 좋을 것 같습니다.